clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_workflows.py

```diff
@@ -1,4 +1,11 @@
-def eddy_fsl_pipeline(low_bval, use_cuda, initrand, name="eddy_fsl"):
+def eddy_fsl_pipeline(
+    low_bval,
+    use_cuda,
+    initrand,
+    image_id: bool = False,
+    field: bool = False,
+    name="eddy_fsl",
+):
```

Comment:
while you're at it, should you precise typing, and give a proper docstring to this function. You may as well want to do the same renaming you've started. If I remember correctly, you weren't fond of the name `low_bval`

Reply:
Names were changed in [32051ae](https://github.com/aramis-lab/clinica/pull/893/commits/32051ae4af21f3031ce13ab7ac6593ea5885a62e)